compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraToMapLibre.kt

```diff
@@ -21,6 +22,18 @@ internal fun MapViewCamera.toCameraPosition(): CameraPosition {
           .tilt(this.state.pitch)
           .bearing(this.state.direction)
     }
+    is CameraState.BoundingBox -> {
+      return map.getCameraForLatLngBounds(
+          this.state.bounds,
+          intArrayOf(
+              this.padding.top.toInt(),
+              // TODO: Internally the C++ lib uses left/right, not start/end
+              this.padding.start.toInt(),
+              this.padding.bottom.toInt(),
+              this.padding.end.toInt()),
```

Comment:
I'm not sure if we need to do something here or not... the C++ layer of course has no concept of localized layouts. We sorta localized our insets. Should the padding behavior depend on this?

Reply:
One other thing I noticed is that padding does not seem to have any effect on Android and I'm not sure why. I have looked at the C++ impl: https://github.com/maplibre/maplibre-native/blob/e6e8cb9d5145034e3856b7c0e183fb93f54fdda3/platform/android/MapboxGLAndroidSDK/src/cpp/native_map_view.cpp#L424. It looks like the C++ is using doubles, but the JNI accepts a list of ints? Not really sure what's up there or if it's a problem 🤷‍♂️

Reply:
Update: I figured out the padding. The PR is now correct (ignoring the question of start/end) and I've added a test that asserts the findings.